game/templates/game/board.html

```
@@ -434,6 +434,9 @@
                     <div id="flipControls" style="display: flex; flex-direction: column; gap: 8px;">
                         <button class="btn btn-gold" id="autoFlipBtn">Auto-Flip: OFF</button>
                     </div>
+                    <button class="btn btn-gold" id="drawBtn" style="display: none;">Offer Draw</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)